var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -126,6 +126,8 @@ def cmake_args(self):
                 '-DTPL_ENABLE_MPI:BOOL=ON',
                 '-DMPI_BASE_DIR:PATH=%s'      % spec['mpi'].prefix,
             ])
+        else:
+            '-DTPL_ENABLE_MPI:BOOL=OFF',
```

Comment:
This trailing comma looks odd to me but I wouldn't be shocked if it is somehow valid Python - have you successfully installed with `~mpi` since making this change?

Reply:
Yes, I did build it successfully. However, the stray comma is likely a
"copy --> paste --> (in)complete edit" error and does not belong there. I just added a commit to fix it.